### DISCUSSION TITLE
using 4.0.5, The settings aren't reflected, and it keeps resetting.

### DISCUSSION THREAD

**Discussion by 1Morganmore** [transmission/transmission#6560] [Q&A]
using 4.0.5, The settings aren't reflected, and it keeps resetting.
How can I solve this issue?

**Comment by Pentaphon** [comment]
Are you on Windows 10 or 11?

Related issue: https://github.com/transmission/transmission/issues/6472

**Comment by 1Morganmore** [comment]
yes

2024년 2월 1일 (목) 오전 9:22, Pentaphon ***@***.***>님이 작성:
